movie-showcase/src/routes/display/+page.svelte

```diff
@@ -0,0 +1,86 @@
+<script lang="ts">
+    import Carousel from 'svelte-carousel'	
+    import { writableArray } from '../../stores';
+    import { slideDuration } from '../../stores';
+    import { carouselItemCount } from '../../stores';    
+    import { onMount } from "svelte";
+
+    onMount(checkSettings);
+
+    function checkSettings(){
+        if($slideDuration == 0 || $slideDuration == null) $slideDuration = 5;
+        if($carouselItemCount == 0 || $carouselItemCount == null) $carouselItemCount = 1;
+    }
+</script>
+
+{#if $writableArray.length >= 1} 
+    <Carousel
+    particlesToShow={$carouselItemCount}
+    autoplay
+    autoplayDuration={0}
+    duration={$slideDuration * 1000}
+    timingFunction="linear"
+    dots={false}
+    arrows={false}
+    swiping={false}
+    >
+
+    {#each $writableArray as item}
+    <div id="card">
+        <li id="poster">
+            <img alt="poster" src={item.Poster}/>
+        </li>
+        <div id="moviedescription">
+                <li id="title">
+                {item.Title}
+                </li>
+                <li>
+                {item.Year} -
+                </li>
+                <li>
+                {item.Plot}
+                </li>
```

Comment:
these list items are missing a `<ul>` wrapper, it that intended ?

Reply:
I removed it due to it conflicting with the carousel styling, was my intention to change them from being list items to just divs 🙂 will be changed in next commit